src/Console/Commands/MakeUser.php

```diff
@@ -164,6 +169,19 @@ protected function emailValidationFails()
         return $this->validationFails($this->email, ['required', new EmailAvailable, 'email']);
     }
 
+    /**
+     * Check if password validation fails.
+     *
+     * @return bool
+     */
+    protected function passwordValidationFails()
+    {
+        return $this->validationFails(
+            $this->data['password'],
+            ['required', PasswordDefaults::rules()],
```

Comment:
One more, sorry. Remove the comma at the end of this line. Older PHP versions don't like it.